addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -36,13 +36,25 @@
                 <span t-if="store.settings.use_push_to_talk and !isMobileOS and !pttExtService.isEnabled" class="small text-muted fst-italic mb-3" t-out="pttExtService.downloadText"/>
             </div>
             <div class="d-flex flex-column">
-                <label t-if="!store.settings.use_push_to_talk" class="d-flex flex-column flex-wrap flex-grow-1 align-items-start" title="Voice detection threshold" aria-label="Voice detection threshold">
+                <t t-if="!store.settings.use_push_to_talk">
                     <span class="me-2 my-1 text-truncate text-wrap">Voice detection threshold</span>
```

Comment:
Slightly out-of-scope of PR, and nitpick: "threshold" feels too technical, "sensitivity" might be more appropriate.
Also thanks to audio-track while testing, we could have a toggle button to automatically set the threshold while testing. This could make it easier to have it well configured for people using the "Voice Detection" option (this can be done eventually in a future PR).

Reply:
I will experiment with that in a following PR, need to make some kind of heuristic to determine what is "active" and what is "background noise" over the duration of the test (like some deviation from mean amplitude).